Tests_v3/MockEnpointResponses/Devices.json

```diff
@@ -0,0 +1,9 @@
+{
+    "devices" : [
+        {
+            "id" : "2552c2ec3ba609bd6ebe5d437e6926de20fde3adae2d2e7f0a5a72bc90285fa5",
+            "push_provider" : "apn"
```

Comment:
Do you think we should serialize this field too? Users can have android and iOS devices in the same account and it could be nice to show this info there.

Reply:
I think it's not needed, at least for now. We can add it once iOS SDK supports FCM IMHO